end-to-end-tests/specs/home.spec.js

```diff
@@ -0,0 +1,93 @@
+var assert = require('assert');
+
+const CBIOPORTAL_URL = process.env.CBIOPORTAL_URL.replace(/\/$/, "");
+
+describe('homepage', function() {
+
+    it('it should show dev mode when testing', function() {
+        browser.url(CBIOPORTAL_URL);
+
+        browser.localStorage('POST', {key: 'localdev', value: 'true'});
+        browser.refresh();
+
+        var devMode = $('.alert-warning');
+
+        devMode.waitForExist(30000);
+        assert(browser.getText('.alert-warning').indexOf('dev mode') > 0);
+    });
+
+    it('it should have 29 studies in list', function () {
+        browser.url(CBIOPORTAL_URL);
```

Comment:
one thing i'm not sure about: if browser is already at this url, does it reload?  we should figure that out because it cost a lot of time to reload for every test.  i guess we can just watch the tests run.